### DISCUSSION TITLE
Help me to fix newpipe fail to download anything from my divece am using the latest version but fail to download always reloaded t

### DISCUSSION THREAD

**Discussion by edwin-1414** [TeamNewPipe/NewPipe#11856] [Questions]
### Checklist

- [X] I made sure that there are *no existing issues or discussions* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have read the [FAQ](https://newpipe.net/FAQ/) and my question isn't listed.
- [X] I have taken the time to fill in all the required details. I understand that the question will be dismissed otherwise.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### What is/are your question(s)?

Help me to fix newpipe fail to download anything from my divece am using the latest version but fail to download always reloaded t

**Comment by asandikci** [comment]
Could you please share crash log

Probably Duplicate of #11803
